app/controllers/catalog_controller.rb

```diff
@@ -98,10 +98,11 @@ def self.uploaded_field
     config.add_facet_field solr_name("subject", :facetable), limit: 5
     config.add_facet_field solr_name("language", :facetable), limit: 5
     config.add_facet_field solr_name("based_near_label", :facetable), limit: 5
+    config.add_facet_field solr_name("part", :facetable), limit: 5, label: 'Part'
     config.add_facet_field solr_name("part_of", :facetable), limit: 5
     # config.add_facet_field solr_name("file_format", :facetable), limit: 5
     # config.add_facet_field solr_name("contributor", :facetable), label: "Contributor", limit: 5
-    config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
+    config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collection'
```

Comment:
Do you know why this changed?

Reply:
It should not have been changed and was fixed.